.node-version

```diff
@@ -1 +1 @@
-v14.21.3
+v18.15.0
```

Comment:
Do we already have a ticket to delete `.node-version` files from the repo, update `.gitignore`, and add `engines` prop to `package.json`?

Reply:
Yes we have a ticket for that. This is the change I made before creating the ticket, so the merge includes it.